src/components/v2/appDetails/k8Resource/nodeDetail/NodeDetail.component.tsx

```diff
@@ -38,11 +38,22 @@ function NodeDetailComponent({
     const [selectedTabName, setSelectedTabName] = useState('')
     const [resourceContainers, setResourceContainers] = useState([])
     const [isResourceDeleted, setResourceDeleted] = useState(false)
+    const [isManagedFields, setManagedFields] = useState(false)
+    const [hideManagedFields, setHideManagedFields] = useState(false)
     const [fetchingResource, setFetchingResource] = useState(
         isResourceBrowserView && params.nodeType === Nodes.Pod.toLowerCase(),
     )
     const { path, url } = useRouteMatch()
+    const toggleManagedFields = (managedFieldsExist: boolean) => {
```

Comment:
no need to pass managedFiledsExist inside the function as it is globally available. directly use isManagedFields inside if condition